pkgs/by-name/lt/ltspice/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  msitools,
+  icoutils,
+  imagemagick,
+  wine64,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "ltspice";
+  version = "24.1.10";
+  src = fetchurl {
+    url = "https://ltspice.analog.com/software/LTspice64.msi";
```

Comment:
```suggestion
    url = "https://web.archive.org/web/20250628214512/https://ltspice.analog.com/software/LTspice64.msi";
```

Reply:
turns out the version from archive.org is outdated, but i don't think it makes any difference as the only change was some model updates.